spec/formatting.md

```diff
@@ -173,13 +174,19 @@ and different implementations MAY choose to perform different levels of resoluti
 >   did not emit an error.
 > - Using a _variable_, the _resolved value_ of an _expression_
 >   could be used as an _operand_ or _option value_ if
->   calling the `getValue()` method of its _resolved value_ did not emit an error.
+>   calling the `getResult()` method of its _resolved value_ did not emit an error.
 >   In this use case, the `resolvedOptions()` method could also
 >   provide a set of option values that could be taken into account by the called function.
+>   - In some cases (such as in a number formatting function), `getResult()` would
+>     return the _operand_ of the function that returned the `MessageValue`. In other
+>     cases (such as in a function that extracts a field from a data structure),
+>     `getResult()` would return a different result than the _operand_.
+> - The `directionality()`, `isolate()`, and `isLiteralOptionValue()` methods
+>   fulfill the requirements and recommendations mentioned earlier in this section.
```

Comment:
"earlier" is incorrect, as at least isolation is only mentioned in bidi handling.

```suggestion
> - The `directionality()`, `isolate()`, and `isLiteralOptionValue()` methods
>   fulfill requirements and recommendations mentioned elsewhere in this specification.
```